Project/Assets/Prairie/Framework/Script/Interaction/Swivel.cs

```diff
@@ -65,7 +65,7 @@ protected void Rotate()
 
 	override public string defaultPrompt {
 		get {
-			return "Open Door";
+			return "Click to Open/Close Door";
```

Comment:
I think we're getting rid of "Click to" in our prompts. The other default prompts just have the action. Could you change this to "Open/Close door"?